containerize.js

```diff
@@ -114,8 +116,13 @@ function listener(details) {
       // If we have a sign-in token, hijack this into a container
       if (object.signInToken) {
         let destination = object.destination;
-        if (!destination) {
-          destination = "https://console.aws.amazon.com";
+        if (!originDestination) {
+          if (!object.destination) {
+            destination = "https://console.aws.amazon.com";
```

Comment:
```suggestion
            destination = "https://console.aws.amazon.com";
            if (object.signInFederationLocation.includes("amazonaws-us-gov.com")) {
              destination = "https://console.amazonaws-us-gov.com";
            } else {
              destination = "https://console.aws.amazon.com";
            }
```
This would fix using this for GovCloud.

Reply:
Created https://github.com/pyro2927/AWS_SSO_Containers/pull/25 - but either PR will work fine.